src/versions.rs

```diff
@@ -1,20 +1,17 @@
 /// npm: <https://www.npmjs.com/package/@holochain/tryorama>
-pub const TRYORAMA_VERSION: &str = "^0.18.3";
+pub const TRYORAMA_VERSION: &str = "^0.19.0-dev.1";
 
 /// npm: <https://www.npmjs.com/package/@holochain/client>
-pub const HOLOCHAIN_CLIENT_VERSION: &str = "^0.19.2";
+pub const HOLOCHAIN_CLIENT_VERSION: &str = "^0.20.0-dev.2";
 
 /// npm: <https://www.npmjs.com/package/@holochain/hc-spin>
 pub const HC_SPIN_VERSION: &str = "^0.500.3";
 
-/// npm: <https://www.npmjs.com/package/@holo-host/web-sdk>
-pub const WEB_SDK_VERSION: &str = "^0.6.20-prerelease";
-
 /// crates.io <https://crates.io/crates/hdi/versions>
-pub const HDI_VERSION: &str = "0.6.5";
+pub const HDI_VERSION: &str = "0.7.0-dev.17";
 
 /// crates.io <https://crates.io/crates/hdk/versions>
-pub const HDK_VERSION: &str = "0.5.5";
+pub const HDK_VERSION: &str = "0.6.0-dev.20";
 
 /// crates.io <https://crates.io/crates/holochain/versions>
-pub const HOLOCHAIN_VERSION: &str = "0.5.5";
+pub const HOLOCHAIN_VERSION: &str = "0.6.0-dev.25";
```

Comment:
```suggestion
pub const HOLOCHAIN_VERSION: &str = "0.6.0-dev.27";
```

Reply:
Was accurate when I put up the PR, updated